From 9f2efa7bc6969c10562ac2c720d50ff77083e5c2 Mon Sep 17 00:00:00 2001
From: Roman Vasiyarov <rvasiyarov@gmail.com>
Date: Mon, 25 Apr 2011 21:34:23 +0400
Subject: [PATCH 5/8] cpufrequtils sysfs: increase MAX_LINE_LEN

larger sysfs data (>255 bytes) was truncated and thus used improperly

Signed-off-by: Roman Vasiyarov <rvasiyarov@gmail.com>
Signed-off-by: Dominik Brodowski <linux@dominikbrodowski.net>
---
 lib/sysfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/sysfs.c b/lib/sysfs.c
index 4e0edab..24dd563 100644
--- a/lib/sysfs.c
+++ b/lib/sysfs.c
@@ -18,7 +18,7 @@
 #include "cpufreq.h"
 
 #define PATH_TO_CPU "/sys/devices/system/cpu/"
-#define MAX_LINE_LEN 255
+#define MAX_LINE_LEN 4096
 #define SYSFS_PATH_MAX 255
 
 /* helper function to read file from /sys into given buffer */
-- 
1.7.10

